llvm/lib/LTO/LTO.cpp

```diff
@@ -158,7 +158,7 @@ void llvm::computeLTOCacheKey(
 
   std::vector<uint64_t> ExportsGUID;
   ExportsGUID.reserve(ExportList.size());
-  for (const auto &VI : ExportList) {
+  for (const auto &[VI, UnusedImportType] : ExportList) {
```

Comment:
We should probably include the new import type result in the cache key. Because if that changes then presumably the cached object should be invalidated as it would be different?